### PR TITLE
Fix broken next section link

### PR DESCRIPTION
**Pull request by MorayM**

### CODE REVIEW

File: GLAA.Web/Views/PrincipalAuthority/PrincipalAuthority.27.cshtml [hunk 1]
```diff
@@ -16,7 +16,7 @@
             </h1>
             @Html.DisplayFor(m => m)
             <p>
-                @Html.ActionLink("Go to next section", "Part", "AlternativeBusinessRepresentatives", 1, new { @class = "button" })
+                @Html.ActionLink("Go to next section", "Part", "AlternativeBusinessRepresentatives", new { id = 1 }, new { @class = "button" })
```
Comment (dougajmcdonald) [inline GLAA.Web/Views/PrincipalAuthority/PrincipalAuthority.27.cshtml:19]: Yup, I saw this in the ABR section too :)

**Review by dougajmcdonald — APPROVED** [review]
Good stuff

File: GLAA.Web/Views/PrincipalAuthority/PrincipalAuthority.27.cshtml (hunk 1 above)
Comment (MorayM) [reply GLAA.Web/Views/PrincipalAuthority/PrincipalAuthority.27.cshtml:19]: Thought it might be in more places - glad you spotted it too!

**Comment by MorayM** [comment]
@dougajmcdonald are you able to give drone a poke? I can't seem to do it from here.

**Pull request merged**